oauth2client/_pure_python_crypt.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Pure python crypto-related routines for oauth2client."""
+
```

Comment:
Always two blank lines before any top-level `class` or `def` and never two blank lines anywhere else ever: https://google.github.io/styleguide/pyguide.html?showone=Blank_Lines#Blank_Lines.